scipy/stats/_stats_py.py

```diff
@@ -507,7 +507,7 @@ def mode(a, axis=0, nan_policy='propagate', keepdims=False):
         message = ("Argument `a` is not recognized as numeric. "
                    "Support for input that cannot be coerced to a numeric "
                    "array was deprecated in SciPy 1.9.0 and removed in SciPy "
-                   "1.11.0. Please consider `pandas.DataFrame.mode`.")
+                   "1.11.0. Please consider `np.unique`.")
```

Comment:
It may be useful to expand on this slightly, as discussed in person, to specify usage of `unique_counts` -- I hadn't actually understood what you were getting at here right away.